update_prevalence.py

```diff
@@ -235,6 +288,66 @@ def cases_week_before(self) -> int:
     def app_key(self) -> str:
         ...
 
+    def set_total_vaccines(self, partial_vaccinations: int, complete_vaccinations: int):
+        self.vaccines_total.partial_vaccinations = partial_vaccinations
+        self.vaccines_total.completed_vaccinations = complete_vaccinations
+    
+    def set_vaccines_of_type(self, vaccine_type: str, partial: int, complete: int):
+        if self.vaccines_by_type is None:
+            self.vaccines_by_type = {}
+        
+        self.vaccines_by_type[vaccine_type] = Vaccination()
+        self.vaccines_by_type[vaccine_type].partial_vaccinations = partial
+        self.vaccines_by_type[vaccine_type].completed_vaccinations = complete
```

Comment:
I think these should unset the other: `set_total_vaccines` should clear `vaccines_by_type` and `set_vaccines_of_type` should clear `vaccines_total`.

Reply:
That would not work as intended - CAN and JHU both provide state data, but CAN only provides total vaccines while JHU provides vaccine breakdowns. We injest JHU first, so we don't want to override that when adding CAN.

I'd rather just store both and then use the best data when processing. Left a comment on the fields.